VirtoCommerce.Platform.Core/ModuleFileCollector/ScriptCollector.cs

```diff
@@ -35,12 +34,7 @@ internal override InternalModuleFile[] LoadModulesTargetPaths()
                     continue;
                 }
 
-                var moduleRootFolderName = scriptsMetadata.VirtualPath.Split("/").First();
-                var moduleFolderName = includedModule.Assembly.GetName().Name;
-
-                var scriptsFolderName = scriptsMetadata.VirtualPath.GetRelativeFilePath(moduleRootFolderName, moduleFolderName);
-
-                var targetPath = Path.Join(includedModule.FullPhysicalPath, scriptsFolderName);
+                var targetPath = Path.Join(includedModule.FullPhysicalPath, scriptsMetadata.VirtualPath.Replace("/", "\\"));
```

Comment:
This is not correct. Paths On Linux use `/` as a separator.

Reply:
Okay, i use Path.DirectorySeparatorChar. Thanks.